src/main/java/htsjdk/samtools/util/IOUtil.java

```diff
@@ -1101,6 +1181,20 @@ public static Path getPath(String uriString) throws IOException {
         }
     }
 
+    public static List<Path> getPaths(List<String> uriStrings) throws RuntimeException {
+        return uriStrings.stream().map(s -> {
+            try {
+                return IOUtil.getPath(s);
+            } catch (IOException e) {
+                throw new RuntimeException(e);
+            }
+        }).collect(Collectors.toList());
+    }
+
+    public static List<Path> promoteFiles(List<File> files){
```

Comment:
This needs a better name.  maybe `filesToPaths`?   It's also untested and undocumented.

Reply:
done.